modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -68,7 +68,7 @@ service google-fluentd restart
 logrotate /etc/logrotate.conf
 
 # Change the access level of configs/ rules/ and run_forseti.sh
-chmod -R ug+rwx ${forseti_home}/configs ${forseti_home}/rules ${forseti_home}/install/gcp/scripts/run_forseti.sh
+chmod -R ug+rwx ${forseti_home}/configs ${forseti_home}/rules  ${storage_bucket_name}/scripts/run_forseti.sh
```

Comment:
Does this work? I don't think the 3rd file will be a valid path `${storage_bucket_name}/scripts/run_forseti.sh`.

Reply:
Updated. PTAL